code/backtracking/src/number_of_ways_in_maze/number_of_ways_in_maze.cpp

```diff
@@ -22,18 +22,18 @@ bool solveMaze(char maze[][10],int sol[][10],int i,int j,int m,int n,int &ways){
     }
 
     ///Rec Case
-    ///Assume jis cell pe khada hai vaha se rasta hai
+    ///Assume there is a way from this cell
     sol[i][j] = 1;
 
-    /// Right mein X nahi hai
+    ///Try going Right
     if(j+1<=n && maze[i][j+1]!='X'){
 
         bool rightSeRastaHai = solveMaze(maze,sol,i,j+1,m,n,ways);
```

Comment:
Unbelievable. Even the variable names are in Hindi. Could you try some renaming?

Reply:
Sure. Done.

Reply:
Btw you can understand Hindi too?

Reply:
@twisted-sres Yes, I do speak Hindi, although not as much since I live in the United States.

Reply:
@twisted-sres Yeah, but only somewhat.